src/ekore/anomalous_dimensions/unpolarized/space_like/as4/fhmruvv_approximations/ggg.py

```diff
@@ -0,0 +1,130 @@
+    B4gluon = 68587.64 - 18143.983 * nf + 423.81135 * nf2 + 9.0672154 * 0.1 * nf3
+
+    Ccoeff = 8.5814120 * 10**4 - 1.3880515 * 10**4 * nf + 1.3511111 * 10**2 * nf2
+    Dcoeff = 5.4482808 * 10**4 - 4.3411337 * 10**3 * nf - 2.1333333 * 10 * nf2
+
+    # The known coefficients of 1/x*ln^a x terms, a = 3,2
+    bfkl0 = -8.308617314 * 10**3
+    bfkl1 = -1.069119905 * 10**5 - 9.963830436 * 10**2 * nf
+
+    #  The resulting part of the function
+    P3gg01 = (
+        +bfkl0 * (-(6 / (-1 + n) ** 4))
+        + bfkl1 * 2 / (-1 + n) ** 3
+        + A4gluon * (-S1)
+        + B4gluon
+        + Ccoeff * lm11(n, S1)
+        + Dcoeff * 1 / n
+    )
+
+    # The selected approximations for nf = 3, 4, 5
+    if nf == 3:
+        P3ggApp1 = (
+            P3gg01
+            + 3.4 * bfkl1 * -(1 / (-1 + n) ** 2)
+            - 345063.0 * 1 / ((-1 + n) * n)
+            + 86650.0 * (1 / n - 1 / (1 + n) + 1 / (2 + n) - 1 / (3 + n))
+            + 158160.0 * (-(1 / n**2))
+            - 15741.0 * lm12m1(n, S1, S2)
+            - 9417.0 * lm13m1(n, S1, S2, S3)
+        )
+        P3ggApp2 = (
+            P3gg01
+            + 5.4 * bfkl1 * -(1 / (-1 + n) ** 2)
+            - 1265632.0 * 1 / ((-1 + n) * n)
+            - 656644.0 * (1 / n - 1 / (1 + n) + 1 / (2 + n) - 1 / (3 + n))
+            - 1352233.0 * (-(1 / n**2))
+            + 203298.0 * lm12m1(n, S1, S2)
+            + 39112.0 * lm13m1(n, S1, S2, S3)
+        )
+    elif nf == 4:
+        P3ggApp1 = (
+            P3gg01
+            + 3.4 * bfkl1 * -(1 / (-1 + n) ** 2)
+            - 342625.0 * 1 / ((-1 + n) * n)
+            + 100372.0 * (1 / n - 1 / (1 + n) + 1 / (2 + n) - 1 / (3 + n))
+            + 189167.0 * (-(1 / n**2))
+            - 29762.0 * lm12m1(n, S1, S2)
+            - 12102.0 * lm13m1(n, S1, S2, S3)
+        )
+        P3ggApp2 = (
+            P3gg01
+            + 5.4 * bfkl1 * -(1 / (-1 + n) ** 2)
+            - 1271540.0 * 1 / ((-1 + n) * n)
+            - 649661.0 * (1 / n - 1 / (1 + n) + 1 / (2 + n) - 1 / (3 + n))
+            - 1334919.0 * (-(1 / n**2))
+            + 191263.0 * lm12m1(n, S1, S2)
+            + 36867.0 * lm13m1(n, S1, S2, S3)
+        )
+    elif nf == 5:
+        P3ggApp1 = (
+            P3gg01
+            + 3.4 * bfkl1 * -(1 / (-1 + n) ** 2)
+            - 337540.0 * 1 / ((-1 + n) * n)
+            + 119366.0 * (1 / n - 1 / (1 + n) + 1 / (2 + n) - 1 / (3 + n))
+            + 223769.0 * (-(1 / n**2))
+            - 45129.0 * lm12m1(n, S1, S2)
+            - 15046.0 * lm13m1(n, S1, S2, S3)
+        )
+        P3ggApp2 = (
+            P3gg01
+            + 5.4 * bfkl1 * -(1 / (-1 + n) ** 2)
+            - 1274800.0 * 1 / ((-1 + n) * n)
+            - 637406.0 * (1 / n - 1 / (1 + n) + 1 / (2 + n) - 1 / (3 + n))
+            - 1314010.0 * (-(1 / n**2))
+            + 177882.0 * lm12m1(n, S1, S2)
+            + 34362.0 * lm13m1(n, S1, S2, S3)
+        )
+    else:
+        raise NotImplementedError("nf^6 is not available at N3LO")
```

Comment:
```suggestion
        raise NotImplementedError("nf=6 is not available at N3LO")
```
likewise for the others ($n_f^k$ actually appears somewhere else)